exporter/kafkaexporter/README.md

```diff
@@ -12,6 +12,7 @@ The following settings can be optionally configured:
 - `topic` (default = otlp_spans for traces, otlp_metrics for metrics, otlp_logs for logs): The name of the kafka topic to export to.
 - `encoding` (default = otlp_proto): The encoding of the traces sent to kafka. All available encodings:
   - `otlp_proto`: payload is Protobuf serialized from `ExportTraceServiceRequest` if set as a traces exporter or `ExportMetricsServiceRequest` for metrics or `ExportLogsServiceRequest` for logs.
+  - `otlp_json`: payload is JSON serialized and follows the same as `otlp_proto`.
```

Comment:
Follows what? Looks like there is a missing word.

Reply:
Corrected this one :)